### DISCUSSION TITLE
i couldn't register and login to my account.

### DISCUSSION THREAD

**Discussion by BoburBerdiev** [vercel/community#3695] [Account Issue] [answered]
### Question

i couldn't to register and login to my account. I also try to create new account then also i got eror "try after 12 hours". please fix it . and i also try to login with my github and it doesn't logins. Please fix it or give me the documentation

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
